modules/standard/OS.chpl

```diff
@@ -1298,53 +1328,135 @@ module OS {
   }
 
   /*
-     :class:`IoError` is the subclass of :class:`SystemError` that serves as the
-     base class for all errors regarding inputs and their formatting.
-     They are typically not directly generated by the OS, but they are
-     used and emitted by the IO module.
+     :class:`IoError` is the subclass of :class:`SystemError` corresponding to
+     EIO.
```

Comment:
I would still keep something about this being the base class for other IO error classes.

Reply:
`IoError` is *not* the base class for other IO error classes, for two reasons:

- It subclasses `SystemError`, so if it was a base class for the Chapel-specific error classes their error codes would be exposed defeating the purpose of this PR.
- It corresponds to `EIO` which generally would be a system IO error.

Reply:
Ok, makes sense. I was asking because it looks like `InsufficientCapacityError` is still a subclass of `IoError`; however since it's only used once in IO I think this change is ok.

Reply:
Oh, I didn't realize that. But yeah, I'm also ok with it staying that way.